README.md

```diff
@@ -104,9 +104,32 @@ http.createServer(app).listen(3000);
   *    `response`       - `[true | false]`, true by default. To write output directly to response instead of to a file.
   *    `error`          - A function to be called when something goes wrong.
   *    `maxAge`         - MaxAge to be passed in Cache-Control header.
+  *    `log`            - A function used to log data instead of the default one.
```

Comment:
error above probably got missed, but I think the expected type helps
```
`function(key, val)`, used to log data instead of the default `console.error`
```
Not sure if that makes sense as a prefix